demo/button/button-icon.html

```diff
@@ -4,7 +4,7 @@
 	<link rel="stylesheet" href="../styles.css" type="text/css">
 	<script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 	<script type="module">
-		import 'demo-transfigurator/demo-transfigurator.js';
+		import '../../tools/demo-transfigurator/demo-transfigurator.js';
```

Comment:
Thoughts about actually putting this in `components`, since it is one? We could have a `demo` directory with eventually a few components in there.

Reply:
That's certainly possible.  I don't really have a preference - I will certainly move it if that's what we lean towards.  ?